lib/nsxv.py

```diff
@@ -326,14 +329,25 @@ def __str__(self):
     for s in services:
       service = '%s%s' % (service, s)
 
+    # applied_to
+    applied_to_list = ''
+    if self.applied_to:
+      applied_to_list = '<appliedToList>'
+      applied_to_element = '%s%s%s' % (_XML_TABLE.get('appliedToStart'),
+                              self.applied_to,
+                              _XML_TABLE.get('appliedToEnd'))
```

Comment:
Done. Can you please let me know which editor do you use? I have used PyCharm. Also is there any template for the code formatting? Next time I can make sure these indentation is correct.

Thank you